cmd/object-api-interface.go

```diff
@@ -62,4 +64,10 @@ type ObjectLayer interface {
 	// Locking operations
 	ListLocks(bucket, prefix string, duration time.Duration) ([]VolumeLockInfo, error)
 	ClearLocks([]VolumeLockInfo) error
+
+	// Policy operations
+	SetBucketPolicies(string, policy.BucketAccessPolicy) error
+	GetBucketPolicies(string) (policy.BucketAccessPolicy, error)
+	RefreshBucketPolicy(string) error
+	DeleteBucketPolicies(string) error
```

Comment:
We can rename 
SetBucketPolicies to SetBucketPolicy
Similarly rename GetBucketPolicies RefreshBucketPolicy DeleteBucketPolicies

Reply:
done

Reply:
done.